app/models/video.rb

```diff
@@ -14,7 +17,9 @@ def self.search_by_title(search_terms)
   end
 
   def average_rating
-    (reviews.select('rating').map{ |r| r.rating.to_f }.inject('+') / reviews.count).round(1)
+    if reviews.any?
+      (reviews.select('rating').map{ |r| r.rating.to_f }.inject('+') / reviews.count).round(1)
```

Comment:
Rather than this - you could benefit from ActiveRecords#average method :) check it out http://api.rubyonrails.org/classes/ActiveRecord/Calculations.html#method-i-average